test/crypto/bls/main.go

```diff
@@ -13,9 +17,17 @@ func main() {
 
 	startTime := time.Now()
 	for i := 0; i < 1000; i++ {
-		bls.Init(bls.BLS12_381)
 		var sec bls.SecretKey
 		sec.SetByCSPRNG()
+
+		if i == 0 {
+			testECKey, _ := ecdsa.GenerateKey(crypto.S256(), rand.Reader)
+			log.Printf("Secret Key: 0x%s", sec.GetHexString())
+			log.Printf("Secret Key: 0x%s", hex.EncodeToString(sec.GetLittleEndian()))
+			log.Printf("Secret Key Length: %d", len(sec.GetLittleEndian()))
+			log.Printf("Secret Key: 0x%s", hex.EncodeToString(testECKey.D.Bytes()))
```

Comment:
what diff between testECKey and sec? one generated by ecdsa and one generated by bls

Reply:
This is just testing code. I am just checking whether they are different. Turn out no difference.